src/gobexport/exporter/config/brk.py

```diff
@@ -475,6 +475,17 @@ class AantekeningenExportConfig:
 
 class ZakelijkerechtenCsvFormat(BrkCsvFormat):
 
+    def zrt_belast_met_azt_formatter(self, value):
+        assert value and isinstance(value, str)
+
```

Comment:
Why assert value? isinstance solves the problem already.
You can also add a comment that clarifies the exception